README.md

```diff
@@ -405,20 +411,19 @@ Send a payment initiation request for an online payment. Returns an object with
 
 #### PayRequest - Payment Request
 ```java
-public class PayRequest {
-    private String amount = null;
-    private Boolean showTips = null;
-    private PaymentMethod paymentMethod = null;
-}
-
-public class PaymentMethod {
-  private PaymentMethodType paymentMethodType = null;
+public class PaymentInitiationRequest {
+    private String storeId = "UUID of assigned store";
+    private String orderId = "UUID of order generated by Kody";
+    private String paymentReference = "UUID generated for the request with 'pay_' appended: pay_UUID";
```

Comment:
Are both of these statements true?
I thought the `orderId` was generated by the client for an ECOM payment
Also does the `paymentReference` _have_ to start with `pay_` ? or is that just the convention used in the PHP app?

Reply:
> `payment_reference` - a unique reference for the payment, sent from the client and returned by the server

(see below)

Reply:
For ecom orderId is generated in the client, changing that.

As for payment reference I think it's just a convention ? Not sure